examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/PaywallViewActivity.kt

```diff
@@ -14,6 +17,17 @@ import com.revenuecat.purchases.ui.revenuecatui.PaywallListener
 class PaywallViewActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+
+        val windowInsetsController = ViewCompat.getWindowInsetsController(window.decorView)
+        windowInsetsController?.isAppearanceLightNavigationBars = false
+        windowInsetsController?.isAppearanceLightStatusBars = true
```

Comment:
Hmm so this will hardcode the light/dark mode here right? Should we remove it and leave it to the system to decide?

Reply:
Maybe we can do this only if the paywall doesn't have dark mode colors? https://github.com/RevenueCat/purchases-ios/pull/3184

Reply:
sure, this was just to make it more clear